stdlib/sql.go

```diff
@@ -126,6 +126,12 @@ var (
 	fakeTxConns map[*pgx.Conn]*sql.Tx
 )
 
+// GetDefaultDriver return the driver initialize in the init function
+// and used when register pgx driver
+func GetDefaultDriver() *Driver {
```

Comment:
Return **sql.Driver** rather than ***Driver**

Reply:
Hmm. Not sure about this. Are there any specific reasons one way or another aside from general rules of thumb regarding when to return an interface or not?

Reply:
I followed the default way recommanded in go : Take interface, return structs, I thinks its easier to access specific plublic method